spec/values/cicognara_csv_spec.rb

```diff
@@ -13,12 +13,14 @@
   end
 
   describe "#values", vcr: { cassette_name: "cicognara" } do
-    let(:obj) do
-      obj = FactoryGirl.create :scanned_resource_in_collection, source_metadata_identifier: '2068747'
+    let!(:obj) do
+      obj = FactoryGirl.create :complete_scanned_resource_in_collection, source_metadata_identifier: '2068747'
       obj.apply_remote_metadata
       obj.save!
       obj
     end
+    before do
```

Comment:
Is this a remnant of something that was removed? Or does the empty block trigger something?  If it does, a comment would be good to make it clear.